src/com/belsoft/project/CustomDictionary.java

```diff
@@ -0,0 +1,64 @@
+package com.belsoft.project;
+
+import java.util.Comparator;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CustomDictionary {
```

Comment:
The intention is to implement a generic dictionary, meaning that we expect to use what Java Generics have to offer in order to obtain a general and extensible solution. This solution should allow us to easily store new possible key-value pairs in the dictionary (for example, VIN and car info, IBAN and customer info, student identity number and student info etc).